oat-sa/extension-tao-test-runner-plugins
# [TAO-8161] iOS fullscreen issue

Related to : https://oat-sa.atlassian.net/browse/TAO-8161
 
Full description of changes.
  
#### How to test
 
- Prepare a delivery with secure mode enabled
- Run delivery using iOS device
- Try to discontinue
- Check that there is not fullscreen notification
